src/main/java/com/fauna/annotation/FaunaColl.java

```diff
@@ -0,0 +1,13 @@
+package com.fauna.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Attribute used to indicate that a field should be ignored during encoding and decoding.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface FaunaColl { }
```

Comment:
Does _Coll_ mean Column here? If so I would rather just call it _FaunaColumn_. If not then it's not explained anywhere what _Coll_ means.

Reply:
it means Collection, but I called it Coll because the field is actually `coll`. if someone wants to use it, it will be aligned.

i was worried that FaunaCollection might be confusing for people and maybe they'd misuse it